modules/local/snpEff.nf

```diff
@@ -11,15 +11,14 @@ process snpEff_Build {
   tuple path(genome_fasta), path(gff_file)
 
   output:
-  tuple path("${genome_fasta.simpleName}/snpEff.config"), path("${genome_fasta.simpleName}/data/${genome_fasta.simpleName}")
+  path("data/${genome_fasta.simpleName}")
 
   script:
   """
   mkdir -p data/${genome_fasta.simpleName}
   cp $gff_file data/${genome_fasta.simpleName}/genes.gff
   cp $genome_fasta data/${genome_fasta.simpleName}/sequences.fa
-  echo "${genome_fasta.simpleName}.genome : ${genome_fasta.simpleName}" >> snpEff.config
-  snpEff build -gff3 -v ${genome_fasta.simpleName}
+  java -jar /opt/snpEff/snpEff.jar build -configOption ${genome_fasta.simpleName}.genome=${genome_fasta.simpleName} -dataDir `pwd`/data/ -gff3 -v -noCheckCds -noCheckProtein ${genome_fasta.simpleName}
```

Comment:
Eventually, `java -jar /opt/snpEff/snpEff.jar` should be a `param.snpEff_app`, similar to how we handle `samtools` and other program paths:

https://github.com/isugifNF/GATK-flow/blob/73bfc89127af3341eb0e021d058666ca7327bb4b/nextflow.config#L30-L38